### DISCUSSION TITLE
Hurst Exponent

### DISCUSSION THREAD

**Discussion by DaveSkender** [DaveSkender/Stock.Indicators#477] [Indicators]
Community discussion for [Hurst Exponent](https://daveskender.github.io/Stock.Indicators/indicators/Hurst/#content)

**Comment by sshquack** [comment]
@DaveSkender I'm trying to add the Hurst Exponent to https://charts.stockindicators.dev/ but it doesn't show up as an option. How can I draw that?

**Reply by DaveSkender** [reply]
I just released the update to Nuget in v2.4.7
I haven't added Hurst Exponent to the demo chart site yet, but if you're feeling ambitious, it's just a matter of updating the [metadata](https://github.com/DaveSkender/Stock.Charts/issues/212).

**Comment by eddyelly** [comment]
Am getting some difficulties with hurst exponent. Can you help me?? @DaveSkender

**Reply by DaveSkender** [reply]
I’ll try, do you want to elaborate on the problem in this thread?  You can also privately message me [here](https://twitter.com/messages/compose?recipient_id=27475431)
